app/graphql/mutations/transformers/panoptes_classification.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../geo'
```

Comment:
you shouldn't have to manually require this file if rails autoload paths are setup correctly. no biggie if not